### PR TITLE
create svgdigitizer package

### LINKED ISSUE #4: Add rudimentary CI
We should add a GitHub workflow to check that the digitizer toolchain works on some demo input.

### PR DESCRIPTION
**Pull request by saraedum**
I am trying to clean up the structure of this repository a bit. Mostly by pressing this into a standard python package structure. While I am at it, I am adding some CI and a tiny user interface to the digitizer. (Also, added a crappy logo.)

* [x] enable CI
* [x] create click interface
* [x] add setup.py

Fixes #4.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -6,10 +6,21 @@ on:
...
+        python: ["3.6", "3.7", "3.8", "3.9"]
     steps:
       - uses: actions/checkout@v2
         with: { submodules: recursive }
+      - uses: conda-incubator/setup-miniconda@v2
+        with: { mamba-version: "*", channels: "conda-forge", channel-priority: true }
       - name: install dependencies
         shell: bash -l {0}
         run: |
-          echo Hello World.
```
Comment (saraedum) [inline .github/workflows/test.yml:15]: I had to add a dummy action directly to master for the CI to actually run. This is only a problem the very first time CI is added to a project on github.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -6,10 +6,21 @@ on:
...
+          mamba env update --quiet -n test -f environment.yml
+          conda list
+      - name: cli-test
+        shell: bash -l {0}
+        run: |
+          python -m svgdigitizer digitize test/data/Ni111_NaOH_Beden1985_Fig2c
+          diff test/data/Ni111_NaOH_Beden1985_Fig2c.csv test/data/Ni111_NaOH_Beden1985_Fig2c.csv.expected
```
Comment (saraedum) [inline .github/workflows/test.yml:26]: For every Pull Request, we will run this check, i.e., run the digitizer on a sample file and check that the generated CSV matches the expected output. (This is not a great check but it was the easiest check that I could think of now.)

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,139 @@
+# Byte-compiled / optimized / DLL files
```
Comment (saraedum) [inline .gitignore:1]: This is a predefined list of files to ignore in Python projects from GitHub.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,42 @@
...
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  digitize
+  plot
+$ python -m svgdigitizer plot test/data/Ni111_NaOH_Beden1985_Fig2c.svg
+[displays a plot]
+$ python -m svgdigitizer digitize test/data/Ni111_NaOH_Beden1985_Fig2c
+[creates test/data/Ni111_NaOH_Beden1985_Fig2c.csv]
```
Comment (saraedum) [inline README.md:24]: I added a simple command line interface to the digitizer. It does not take any options yet but maybe it's a good starting point already for those who do not want to digitize from Jupyter notebooks.

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+from distutils.core import setup
+
+setup(
+    name = 'svgdigitizer',
+    version = '0.1.0',
+    packages = ['svgdigitizer',],
+    license = 'GPL 3.0+',
+    long_description = open('README.md').read(),
+    include_package_data=True,
+)
```
Comment (saraedum) [inline setup.py:11]: With this, it should be possible to `pip install .` the svgdigitizer. Once we published it to PyPI, it should be possible to `pip install svgdigitizer`.

File: .github/workflows/test.yml (hunk 2 above)
Comment (saraedum) [reply .github/workflows/test.yml:26]: You can see the result of these tests at the bottom of this Pull Request where it (hopefully) says "All checks have passed".

File: README.md [hunk 6]
```diff
@@ -1,14 +1,48 @@
...
+# SVGDigitizer — Create (x,y) Data Points from SVG files.
+
+![Logo](./logo.svg)
+
+The purpose of this project is to digitize plots in scientific publications, recovering the measured data visualized in such a plot.
+
+## Installation
+
+```
+pip install -e .
```
Comment (saraedum) [inline README.md:10]: The package is now installable with pip. This puts `svgdigitizer` on the path. You might have to open a new shell or type `hash -r` for `svgdigitizer` to actually work initially.

**Review by linuxrider — APPROVED** [review]

**Comment by saraedum** [comment]
@linuxrider thanks for having a look. Let's merge this then.

**Pull request merged**